src/components/ShoppingListItem/ShoppingListItem.js

```diff
@@ -128,8 +131,11 @@ const ShoppingListItem = ({
         </label>
         <button
           type="button"
+          onClick={() => setShowSingleDetail(!showSingleDetail)}
```

Comment:
Here is where I've tried to get the individual panel to open. Note that I've set state showSingleDetail to use `showDetails` from the parent